### ISSUE TITLE
Incompatible with SBT 1.4.x

### ISSUE THREAD

**Issue by Madoc** [kxbmap/sbt-jooq#28]
Trying to run `jooqCodegen` on SBT 1.4.x results in the error:

```
[error] (Compile / jooqCodegenConfigTransformer) java.lang.NoClassDefFoundError: fastparse/core/Implicits$Repeater
```

Setting the SBT version back to 1.3.13 fixes the problem.
Presumably, the fastparse library was either removed or updated in SBT 1.4.0.

**Comment by kxbmap** [comment]
Thank you for the report.
Released version 0.6.0 which fixes the problem.

**Comment by Madoc** [comment]
Thank you! I can confirm it works.
